api/authorization.yml

```diff
@@ -363,6 +366,15 @@ components:
           items:
             $ref: "#/components/schemas/ExternalPrincipal"
 
+    ExternalLoginInformation:
```

Comment:
lakeFS should not be aware of the details of the identity token details fields or params. 
it should pass it to the remote authentication services. 
so in other words, it should be some **generic** object key/value in the body that will be passed to the remote authentication service as is. 

The remote service on the other hand that does the login will seriallize this object and extract specific fields to create AWS request.